packages/console/src/components/Search/index.module.scss

```diff
@@ -4,12 +4,12 @@
   display: flex;
   align-items: center;
 
-  :not(:first-child) {
+  >:not(:first-child) {
     margin-left: _.unit(2);
   }
 
   .searchInput {
-    width: 286px;
+    width: 306px;
```

Comment:
Why do we use a fixed width for the search component?

Reply:
The original search component needs longer to display the placeholder, and the `TextInput` component does not support automatic width adjustment according to the placeholder content.
So follow the original implementation this time.

<img width="384" alt="image" src="https://user-images.githubusercontent.com/10806653/212325949-6ac85794-6702-4d98-b1a2-35acad821d85.png">

Reply:
Finally, I found that the width settings should be specified separately.
Already updated.